tools/register_users_from_csv.py

```diff
@@ -149,12 +193,19 @@ def get_sanitized_name(name):
         "Requires `oc` command and cluster admin access to OpenShift."
     ),
 )
-def main(csv_file, add_to_rhods_notebooks_namespace):
+@click.option(
+    "--auth-type",
```

Comment:
Please set `secret` as a default and rename `device` to `oauth2`.

Reply:
@knikolla I have renamed the auth method as you asked